rpcs3/Emu/VFS.cpp

```diff
@@ -73,6 +73,11 @@ bool vfs::mount(std::string_view vpath, std::string_view path, bool is_dir)
 				list.back()->path += '/';
 			if (!is_dir && list.back()->path.ends_with('/'))
 				vfs_log.error("File mounted with trailing /.");
+			const auto mp = lv2_fs_object::get_mp(vpath_backup);
+			if (mp == &g_mp_sys_dev_usb)
+				mp->is_mounted |= (1U << vpath_backup.back() - '0');
+			else
+				mp->is_mounted = true;
```

Comment:
this is not a boolean

Reply:
I use it to store the mount status of 8 dev_usbXXX devices, for the other devices, it is used as a boolean.

Reply:
I know. that doesn't change the fact though

Reply:
So I should change it to 0 and 1 for devices other than dev_usbXXX?

Reply:
Done.